src/ontodia/data/sparql/sparqlDataProvider.ts

```diff
@@ -248,10 +293,39 @@ export class SparqlDataProvider implements DataProvider {
         }
 
         const {defaultPrefix, fullTextSearch, dataLabelProperty} = this.settings;
+        const blankQueryParameters = this.options.acceptBlankNodes
```

Comment:
Please, move `blankQueryParameters` and `blankNodeQuery` as constants to `blankNodes.ts`;